src/main/java/org/jenkinsci/maven/plugins/hpi/RunMojo.java

```diff
@@ -81,7 +83,7 @@
  * 
  * @author Kohsuke Kawaguchi
  */
-@Mojo(name="run")
+@Mojo(name="run", requiresDependencyResolution=ResolutionScope.TEST)
```

Comment:
Hmm, however there is also the `dependencyResolution` parameter, used at the start of `execute`; and `resolveDependencies("test")` in `getJenkinsWarArtifact`. Yet these do not work. @stephenc any clue what is going on here?